src/instructlab/model/evaluate.py

```diff
@@ -510,6 +510,16 @@ def evaluate(
     enable_serving_output,
 ):
     """Evaluates a trained model"""
+
+    # warn user if GPUs are not configured to be used
+    eval_serve = deepcopy(ctx.obj.config.serve)
+    is_gpus = gpus or eval_serve.vllm.gpus
```

Comment:
To do this, you're going to need the get_attribute method from https://github.com/instructlab/instructlab/pull/2072 so you can check settings for tps as well.

I would also expect the logic to move up where the other GPU checking is taking place.  Or at least before select_backend.

Reply:
Yeah, going with a different approach here - moved the logic elsewhere